Fibonacci/fibo_print_first_n_terms.cpp

```diff
@@ -0,0 +1,55 @@
+#include<iostream>
+uint64_t fibo(int n)
```

Comment:
return type of this function is uint64_t but it is not returning anything and that is causing the hang state at the last. Think whether you really want to return anything or not